plugins/destination/mysql/cloud-config-ui/e2e-tests/MOVE_TO_LIB/plugin-ui-helpers.ts

```diff
@@ -0,0 +1,168 @@
+import cryptoRandomString from 'crypto-random-string';
+import { expect, Frame, Page, test } from '@playwright/test';
+import { click, fillInput, getMainTestUser, getPluginUrl, getRootUrl } from './e2e-helpers';
+
+export const getPersistentName = () => `name-${cryptoRandomString(10)}`;
+
+type CreatePluginControlOpts = {
+  page: Page;
+  kind: 'source' | 'destination';
+  pluginName: string;
+  pluginLabel: string;
+  pluginNewName: string;
+  fillFieldsSteps?: (iframeElement: Frame) => Promise<void>;
+};
+
+interface EditPluginControlOpts extends CreatePluginControlOpts {
+  pluginUrl: string;
+}
+
+export const login = async (page: Page) => {
+  await page.goto('https://cloud.cloudquery.io/auth/login');
+
+  const { email, password } = getMainTestUser();
+
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').fill(email);
```

Comment:
We could use the util methods which you copied into e2e-helpers but if it works without them consistently - I am okay with that

Reply:
handled in the lib, thanks for the point out https://github.com/cloudquery/plugin-config-ui-lib/pull/51/files?w=1#diff-97a4a70134d628365b637c8f2f2c1ed7951322771a8b97559a13932ecaac8a90